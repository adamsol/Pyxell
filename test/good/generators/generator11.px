
func range(a, b): Int... def
    for x in a..b do
        print x
        yield x
    print null, a, b

for x, y, z in range(2, 3), range(1, 1), range(4, 5) do
    print x, y, z
