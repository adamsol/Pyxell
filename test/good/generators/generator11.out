2
1
4
2 1 4
3
null 1 1
